common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -8,7 +8,7 @@
 <form class="choicegroup capa_inputtype" id="inputtype_${id}">
     <fieldset ${describedby}>
       % if response_data['label']:
-        <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${response_data['label']}</legend>
+        <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${HTML(response_data['label'])}</legend>
```

Comment:
@robrap For more context the `response_data['label']` is something defined by course instructors and they have freedom to use anything. It could be text + html. Also there are some places where this value is being set as attribute, for those cases we choose `Text` but not sure what to do in those kind of scenarios.